src/excalidraw-app/index.tsx

```diff
@@ -163,30 +162,10 @@ const initializeScene = async (opts: {
 };
 
 const ExcalidrawWrapper = () => {
-  // dimensions
-  // ---------------------------------------------------------------------------
-
-  const [dimensions, setDimensions] = useState({
-    width: window.innerWidth,
-    height: window.innerHeight,
-  });
   const [errorMessage, setErrorMessage] = useState("");
   const currentLangCode = languageDetector.detect() || defaultLang.code;
   const [langCode, setLangCode] = useState(currentLangCode);
 
-  useLayoutEffect(() => {
-    const onResize = () => {
-      setDimensions({
-        width: window.innerWidth,
-        height: window.innerHeight,
-      });
-    };
-
-    window.addEventListener("resize", onResize);
-
-    return () => window.removeEventListener("resize", onResize);
-  }, []);
-
```

Comment:
do we keep this as a fallback when ResizeObserver isn't supported?

Reply:
I mean putting a comparative logic inside App.tsx that would call update `updateDOMRect`

Reply:
It has good support so shouldn't need that https://caniuse.com/?search=resizeob, and we don't support IE 11 as well correct ?

Reply:
It has 90% support but I guess we can leave it as is and see if anyone complains.